### DISCUSSION TITLE
Nvidia store selling channel via LDLC (FR specific) for FE cards

### DISCUSSION THREAD

**Discussion by singsonn** [jef/streetmerchant#1135] [Help]
### Description

Hi,

Not sure if it was implemented already, but I have a question regarding the new method of selling FE cards by Nvidia, specifically interested in the FR store.

Now, published links to FE cards are being changed on every drop to redirect to LDLC store.

They appears on NV API at api.nvidia.partners at ".searchProduct.featuredProduct.retailers". Example for 3080 FE : https://api.nvidia.partners/edge/product/search?page=1&limit=9&locale=fr-fr&category=GPU&gpu=RTX%20380&manufacturer=NVIDIA&manufacturer_filter=NVIDIA~1,ASUS~1,EVGA~3,GIGABYTE~2,MSI~1,PNY~0,ZOTAC~0 

When published, the json looks like this :
![](https://imgur.com/bT7oI1H.png)

Don't know if it has already been adapted to the "nvidia" or "nvidia-api" stores. I tried looking up the code, but couldn't find an answer to my question.

Thanks for your help.

**Comment by jef** [comment]
It has not been adapted. Feel free to update the `nvidia-api`. It's pretty much deprecated at this point as it's stock is pretty inconsistent across the globe.

Sorry I can't be of much help here. I think the best thing to do if you're looking for an Nvidia card, is to just use `nvidia` and click on the link from there. Otherwise, feel free to take a crack at solving this part.

**Comment by marinovdeyan** [comment]
@jef does `nvidia-api` return the ldlc link when using it in `france`?

**Comment by singsonn** [comment]
Hi @marinovdeyan, yes, it returns the LDLC link when using it for France. You can see it in my screenshot on my first message. Currently, the API does not return a link though.

**Comment by marinovdeyan** [comment]
@singsonn Thanks for your answer. Just to confirm: I wrote a very simple script that refreshes the nvidia partners API 3080 json file every 5 seconds and checks the length of the "retailers" and send me a discord message with the value of retailers when retailers' length is more than 0. Will a link be available there when the 3080 drops on LDLC ?

Thanks in advance for your help.

**Comment by singsonn** [comment]
@marinovdeyan On the last FE drop on LDLC (that was last Friday Dec. 4th), the 3080s went away in minutes. It ran out of stock so fast that the link on the API didn't even come up... I have a bash script running, and no link was detected for the 3080s. The link was detected for the 3070s though, because they sold less quickly. I am doing it a bit different than you, I am scanning for the "prdStatus" parameter of the json to be different than "out_of_stock". But your solution is even better I think.

**Comment by marinovdeyan** [comment]
@singsonn that explains why so many tracking bots didn't detect the 3080s on LDLC on Friday! How did people even buy the product then? Do you know of a way to "solve" this, or should we just hope that they don't sell out that fast. Btw, just a quick tip, if you live in the Geneva region, as said on your profile (which also happens to be my location), the UK FE retailer ([scan.co.uk](url)) ships to France and Switzerland (I'm assuming the delivery to France is much cheaper though), making it in interesting retailer to track too!

**Comment by singsonn** [comment]
@marinovdeyan there is a discord server (in French though) were the owner knew about the drop 15-20 minutes before it happened. I guess he knows someone working at LDLC. He sent the link to the community before the API was up. That's why it went so fast. FYI, here is the discord server in question : https://discord.gg/uZdYYGavbS
I'm not sure Scan is sending to FR, heard cancellation stories from outside UK/IE buyers. I would guess the local distributor has exclusivity for the sale of the FEs.

**Reply by xkuza** [reply]
can you drop a new invite link?

**Reply by singsonn** [reply]
> can you drop a new invite link?

Sure, here you go: https://discord.gg/DQbEqxKGYg

Got the link from his website in case it expires: https://www.bavarnold.com/

**Comment by marinovdeyan** [comment]
@singsonn Wow thank you for the info, will definitely be following the information on the server. Thanks for informing me about scan, they definitely offer you the info at checkout, so I hope those cancellation stories are just exceptions (although it's hard to be an exception, when the whole user base can only buy 10 cards in total :p).

Have a nice day and good luck in finding the world's rarest graphics card!

**Comment by suntmarie** [comment]
could it be that ldlc has a country filter .... i am at the moment in Germany and cant start ldlc scanning. the software do nothing.....
